modules/tile-converter/src/i3s-converter/helpers/feature-attributes.ts

```diff
@@ -42,9 +42,11 @@ export function flattenPropertyTableByFeatureIds(
 function getPropertiesByFeatureIds(properties: any[], featureIds: number[]): any[] {
```

Comment:
Can we use `unknown` instead of `any`
```suggestion
function getPropertiesByFeatureIds(properties: unknown[], featureIds: number[]): unknown[] {
```